packages/extension-base/src/utils/localStorageStores/createStoreDefinition.ts

```diff
@@ -24,7 +22,7 @@ export default <Content>(schema: z.ZodType<Content>, defaultValue?: Content) =>
     const newValueParsingResult = schema.safeParse(changesFromNamespace.newValue);
 
     if (!newValueParsingResult.success) {
-      console.warn(`The new local storage value in namespace "${namespace}" failed to match the namespace schema with error: "${newValueParsingResult.error.toString()}"`);
+      console.error(`The new local storage value in namespace "${namespace}" failed to match the namespace schema with error: "${newValueParsingResult.error.toString()}"`);
```

Comment:
We use `.error` in other cases like this, so to be consistent. Also more proper than warning for it indicates incorrect data in local storage, which is a pathological situation.